src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/TableMappingFactory.java

```diff
@@ -189,14 +189,14 @@ TableMapping getTableMapping(DynamoTableDescription virtualTable) {
                 virtualTableCreationValidator.validateAndGetSecondaryIndexMap(virtualTable, physicalTable);
         TableMapping tableMapping = partitioningStrategy.createTableMapping(virtualTable, physicalTable,
             secondaryIndexMap::get, mtContext);
-        LOG.info("created virtual to physical table mapping: " + tableMapping.toString());
+        LOG.debug("created virtual to physical table mapping: " + tableMapping.toString());
```

Comment:
We should add a guard `if (LOG.isDebugEnabled)` and/or use log templates `LOG.debug("created virtual to physical table mapping: {}", tableMapping.toString());`, so that we don't construct `String`s unnecessarily when debug logging is not enabled.